src/components/interface/PositionTable/components/PositionTableRow/components/CurrentMargin/CurrentMargin.tsx

```diff
@@ -40,22 +39,34 @@ const CurrentMargin: React.FunctionComponent<CurrentMarginProps> = ({
     }
   };
 
-  const getNetMarginLabel = () => (
-    <>
-      Margin
-      {!isUndefined(accruedCashflow) && !isSettled && (
-        <Box
-          component="span"
-          sx={{ color: accruedCashflow >= 0 ? colors.vzCustomGreen1 : colors.vzCustomRed1 }}
+  const getNetMarginLabel = () => {
+    if (isUndefined(accruedCashflow) || isSettled) {
+      return 'Margin';
+    }
+
+    const formattedAccFlow = formatCurrency(Math.abs(accruedCashflow));
+    const accFlowValue = parseFloat(formattedAccFlow);
+
+    return (
+      <>
+        Margin
+        <span
+          style={{
+            color:
+              accFlowValue > 0
+                ? colors.vzCustomGreen1.base
+                : accFlowValue < 0
+                ? colors.vzCustomRed1.base
+                : undefined,
```

Comment:
what's the default color?

Reply:
whatever the parent component will provide, hence undefined here. To respect that style